spec/support/fixtures/test/main.swift

```diff
@@ -1,7 +1,7 @@
 protocol RedundantProtocol {
 }
 
-class SomeClass: RedundantProtocol {
+public class SomeClass: RedundantProtocol {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/manicmaniac/danger-periphery/blob/c77010dc67e9fa3c3ac9d6a49d6c4933a5c62fc1/spec/support/fixtures/test/main.swift#L4"></span>Class 'SomeClass' is declared public, but not used outside of test</td>
    </tr>
  </tbody>
</table>